spec/v2/index.bs

```diff
@@ -191,7 +191,7 @@ below). See [[#api-auth]] for details.
 </figure>
 
 1. The data recipient sends a `ProductFootprintFragment`  to the `Events Endpoint` of the data owner. The data recipient should always include 1 or more product ids in property `productIds`. The data recipient should limit each fragment to exactly 1 specific product (e.g. a specific type of apple instead of all apples that somebody is offering)
-    1. If the recipient is requesting a specific reporting period, it should include the reporting
+    1. If the recipient is requesting a specific reference period, it should include the reporting
```

Comment:
Also the "reporting period" mentioned after the comma should change to "reference period". I counted a total of 11 references to "reporting period" in the tech specs, and this is the one we are missing in this ADR.

Reply:
Sharp! This one should be changed as well of course